packages/lib/event-types/getEventTypesPublic.ts

```diff
@@ -21,42 +23,46 @@ export async function getEventTypesPublic(userId: number) {
   }));
 }
 
+type BaseEventType = Prisma.EventTypeGetPayload<{
+  select: typeof baseEventTypeSelect;
+}>;
+
+type RawEventType = BaseEventType & {
+  metadata: Record<string, any> | null;
+};
+
 const getEventTypesWithHiddenFromDB = async (userId: number) => {
-  const eventTypes = await prisma.eventType.findMany({
-    where: {
-      AND: [
-        {
-          teamId: null,
-        },
-        {
-          OR: [
-            {
-              userId,
-            },
-            {
-              users: {
-                some: {
-                  id: userId,
-                },
-              },
-            },
-          ],
-        },
-      ],
-    },
-    orderBy: [
-      {
-        position: "desc",
-      },
-      {
-        id: "asc",
-      },
-    ],
-    select: {
-      ...baseEventTypeSelect,
-      metadata: true,
-    },
-  });
+  const eventTypes = await prisma.$queryRaw<RawEventType[]>`
+    SELECT data."id", data."title", data."description", data."length", data."schedulingType"::text,
```

Comment:
At some point, we can start using prisma's filesystem imports for these raw queries. Later PR